ipaserver/plugins/dns.py

```diff
@@ -3883,6 +3895,12 @@ def get_options(self):
 
     def pre_callback(self, ldap, dn, entry_attrs, attrs_list, *keys, **options):
         assert isinstance(dn, DN)
+
+        if options.get('structured') and options.get('raw'):
+            raise errors.MutuallyExclusiveError(
+                reason=_("cannot use structured together with raw")
+            )
+
```

Comment:
For dnsrecord_del the raw option is already excluded from the command line. I think that structured should be as well. Neither makes sense because there is nothing remaining to display. Removing the 'dnsrecord.structured_flag' option seems to do the trick.